--- a/configure	2013-04-14 17:28:23.000000000 -0500
+++ b/configure	2013-04-14 17:29:02.000000000 -0500
@@ -28645,13 +28645,13 @@
   fi
 
 
 
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for SSL support in libcurl" >&5
 $as_echo_n "checking for SSL support in libcurl... " >&6; }
-  CURL_SSL=`$CURL_CONFIG --feature | $EGREP SSL`
+  CURL_SSL=`$CURL_CONFIG --feature | grep SSL`
   if test "$CURL_SSL" = "SSL"; then
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 
 $as_echo "#define HAVE_CURL_SSL 1" >>confdefs.h

