--- a/configure	2013-04-14 17:28:23.000000000 -0500
+++ b/configure	2013-04-14 17:29:02.000000000 -0500
@@ -39335,7 +39335,7 @@
 
   { echo "$as_me:$LINENO: checking for SSL support in libcurl" >&5
 echo $ECHO_N "checking for SSL support in libcurl... $ECHO_C" >&6; }
-  CURL_SSL=`$CURL_CONFIG --feature | $EGREP SSL`
+  CURL_SSL=`$CURL_CONFIG --features | grep SSL`
   if test "$CURL_SSL" = "SSL"; then
     { echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6; }

